homeassistant/components/samsungtv/bridge.py

```diff
@@ -606,7 +596,6 @@ def _remote_event(self, event: str, response: Any) -> None:
                         CONF_PORT: ENCRYPTED_WEBSOCKET_PORT,
                     }
                 )
-                self._notify_reload_callback()
```

Comment:
No need to reload anymore since we will auto reload when it changes from the listener after 5s